controller/platform_clients.go

```diff
@@ -0,0 +1,174 @@
+package controller
+
+import (
+	"context"
+	"github.com/fabric8-services/fabric8-wit/configuration"
+	"github.com/fabric8-services/fabric8-wit/errors"
+	"github.com/fabric8-services/fabric8-wit/kubernetes"
+	"github.com/fabric8-services/fabric8-wit/log"
+	"github.com/goadesign/goa"
+	errs "github.com/pkg/errors"
+	"net/url"
+	"os"
+)
+
+// ClientGetter creates an instances of clients used by this controller
+type ClientGetter interface {
+	GetKubeClient(ctx context.Context) (kubernetes.KubeClientInterface, error)
+	GetAndCheckOSIOClient(ctx context.Context) (OpenshiftIOClient, error)
```

Comment:
Should we really be using `osio` further in our identifiers given that there's gonna be changes :)

Reply:
:)